### ISSUE TITLE
Seemingly miscalculated EnC rude edit

### ISSUE THREAD

**Issue by dpoeschl** [dotnet/roslyn#3067]
![image](https://cloud.githubusercontent.com/assets/235241/7789533/18609e14-0219-11e5-8cb2-5712cecefa3a.png)

**Comment by tmat** [comment]
@dpoeschl I don't think this makes the bar for RTM. Nice catch though and we should fix in an update.

**Comment by tmat** [comment]
@ManishJayaswal Moved to 2.0.

**Comment by CyrusNajmabadi** [comment]
I don't repro this.  I'm able to comment out the 'return' and have things work properly:

<img width="614" height="344" alt="Image" src="https://github.com/user-attachments/assets/39b9e1b6-9f7d-41d2-a55d-7d749cea1300" />
